### PR TITLE
Lab1

### PR DESCRIPTION
**Pull request by mizutombo**
In preparation for Lab2 assignment ...
* Added index.js file as app 'starter button' for Lab1 http server app.
* Added test script "start": "npm run test && node index.js" into package.json file.
* Confirmed that code retains its full functionality before moving on to Lab2 assignment.

### CODE REVIEW

**Comment by mizutombo** [comment]
* added back .eslintrc file
* modified .eslintrc rules
* corrected linter errors per new .eslintrc rules, then ran npm test
* now passes all tests, including travis-ci

File: http.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    }));
+    res.end();
+  }
+});
+
+const port = 8080;
+server.listen(port, err => {
+  if(err) console('error', err);
+  else console.log('http server listening on port', port);
+});
```
Comment (GeoffreyEmerson) [inline http.js]: Typically you'll name this file 'server.js' and export the server function.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// establish index.js as 'start button' for this server app
+
+var http = require('./http');
+
+console.log(http('class'));
```
Comment (GeoffreyEmerson) [inline index.js]: Then in this file you'll require the 'server' file and launch the server by calling the '.listen' method on it. That's how it becomes the "start button" for the app. It also creates a separation point where the server is pretty much complete and testable in the 'server.js' file without having to actually launch it.

**Comment by GeoffreyEmerson** [comment]
Please resubmit this PR directed toward your named branch.

**Pull request closed without merging**